doc/source/whatsnew/v0.23.0.txt

```diff
@@ -924,6 +924,9 @@ read the `NumFOCUS blogpost`_ recapping the sprint.
   to functions, methods and classes.
   (:issue:`18941`, :issue:`18948`, :issue:`18973`, :issue:`19017`)
 - Added a reference to :func:`DataFrame.assign` in the concatenate section of the merging documentation (:issue:`18665`)
+- Updated ``to_gbq`` and ``read_gbq`` documentation to reflect changes from
```

Comment:
maybe "documentation" -> "signature and documentation" (as it is not only a doc change ?) 
and move it to "other enhancements" section